apps/wallet/src/ui/app/components/ledger/ImportLedgerAccounts.tsx

```diff
@@ -29,6 +30,13 @@ const mockAccounts = [
 export function ImportLedgerAccounts() {
     const accountsUrl = useNextMenuUrl(true, `/accounts`);
     const navigate = useNavigate();
+    const [suiLedgerClient] = useSuiLedgerClient();
+
+    if (!suiLedgerClient) {
+        // TODO (future improvement): We should detect when a user's Ledger device has disconnected so that
```

Comment:
This should be possible by subscribing in the provider then just removing the client from the context.

Reply:
And then this would just work "magically" because this becomes a reactive value

Reply:
I fixed this, it was easier to do than I thought :)